clang/test/Driver/sycl-spirv-ext.c

```diff
@@ -46,7 +46,8 @@
 // CHECK-DEFAULT-SAME:,+SPV_INTEL_variable_length_array,+SPV_INTEL_fp_fast_math_mode
 // CHECK-DEFAULT-SAME:,+SPV_INTEL_fpga_cluster_attributes,+SPV_INTEL_loop_fuse
 // CHECK-DEFAULT-SAME:,+SPV_INTEL_long_constant_composite
-// CHECK-DEFAULT-SAME:,+SPV_INTEL_fpga_invocation_pipelining_attributes"
+// CHECK-DEFAULT-SAME:,+SPV_INTEL_fpga_invocation_pipelining_attributes
+// CHECK-DEFAULT-SAME:,+SPV_INTEL_fpga_dsp_control"
```

Comment:
I'd even removed quotation mark from check string.

Reply:
On the other hand, it checks that no other extensions are added unexpectedly. So, nevermind.